packages/ts/generator-typescript-plugin-model/src/EntityModelProcessor.ts

```diff
@@ -139,19 +139,14 @@ export class EntityClassModelProcessor extends EntityModelProcessor {
     return this.#processModelClass(entitySchema, this[$entity].id, parent);
   }
 
-  #processClassElements({ required, properties }: ObjectSchema): readonly ClassElement[] {
+  #processClassElements({ properties }: ObjectSchema): readonly ClassElement[] {
     if (!properties) {
       return [];
     }
 
-    const requiredSet = new Set(required);
     return Object.entries(properties).map(([name, schema]) => {
       const type = new ModelSchemaTypeProcessor(schema, this[$dependencies]).process();
-      const args = new ModelSchemaExpressionProcessor(
-        schema,
-        this[$dependencies],
-        (_) => !requiredSet.has(name),
-      ).process();
+      const args = new ModelSchemaExpressionProcessor(schema, this[$dependencies]).process();
```

Comment:
This usage is the only one that sets `checkOptional` parameter for the [`ModelSchemaExpressionProcessor` constructor](https://github.com/vaadin/hilla/blob/main/packages/ts/generator-typescript-plugin-model/src/ModelSchemaProcessor.ts#L226). Since we don't need it anymore, we could drop the constructor parameter as well to make code cleaner.